docs/water_masking.md

```diff
@@ -0,0 +1,88 @@
+# Water Mask
```

Comment:
Do we want to change this to `Water Masking` to match the name of the page (`water_masking`), or vice-versa?

Reply:
Yeah, I'd agree. Let's change it to "Water Masking".